cubical/workers.py

```diff
@@ -111,9 +114,23 @@ def setup_parallelism(ncpu, nworker, nthread, force_serial, affinity, io_affinit
         True if parallel mode is invoked (i.e. workers are to be launched)
     """
     global num_workers
+    global num_threads
+    def set_numba_threading(nthread):
+        try:
+            numba.config.THREADING_LAYER = "safe"
+            @numba.njit(parallel=True)
+            def foo(a, b):
+                return a + b
+            foo(np.arange(5), np.arange(5))
+            return nthread
+        except:
```

Comment:
Unqualified excepts are generally frowned upon. Does this not raise a consistent exception?

Reply:
it raises a massively complex numba exception which I'm not sure how to properly catch. let me see if I can reproduce. I'm just worried this exception interface will change (numba seems constantly evolving)